layouts/partials/docs/outdated_content.html

```diff
@@ -0,0 +1,20 @@
+{{ if ne .Site.Language.Lang "en" }}
+  {{ $currentLastmodUnixtime := .Lastmod.Unix }}
+  {{ $originCurrentLastmodUnixtime := 0 }}
+  {{ $originPageURL := "" }}
+  {{ $originPageTitle := "" }}
+  {{ range .Translations }}
+    {{ if eq .Lang "en" }}
+      {{ $originCurrentLastmodUnixtime = .Lastmod.Unix }}
+      {{ $originPageURL = .RelPermalink  }}
+      {{ $originPageTitle = .Title }}
+      {{ break }}
+    {{ end }}
+  {{ end }}
+
+  {{ if lt $currentLastmodUnixtime $originCurrentLastmodUnixtime }}
+  <div class="alert alert-danger caution callout" role="alert">
```

Comment:
I don't think having a stale page is dangerous.

Reply:
Indeed. You are right.